REVIEW_GUIDELINES.md

```diff
@@ -54,7 +88,26 @@ selector:
   release: {{ .Release.Name }}
 ```
 
-If a chart has multiple components, a `component` label should be added (e. g. `component: server`). The resource name should get the component as suffix (e. g. `name: {{ template "myapp.fullname" . }}-server`). The `component` label must be added to label selectors as well.
+If a chart has multiple components, a `component` label should be added to the selector (see above).
+
+### Persistence labels
+
+### StatefulSet
+
+In case of a `Statefulset`, `spec.volumeClaimTemplates.metadata.labels` must have both `app` and `release` labels, and **must not** contain `chart` label or any label containing a version of the chart, because `spec.volumeClaimTemplates` is immutable.
```

Comment:
I found out that you cannot set `spec.volumeClaimTemplates.metadata.labels`. These labels are automatically set from the StatefulSet's `spec.selector.matchLabels`.

https://github.com/kubernetes/kubernetes/pull/60693
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/statefulset/stateful_set_utils.go#L152

Reply:
Oh, right, so we should not put any labels on volumeClaimTemplate?

Reply:
I'm puzzled here. This is what I do in redis: https://github.com/helm/charts/pull/7686 and it seemed to have an impact (see https://github.com/helm/charts/issues/7803).